services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/extensions/dynamodb/mappingclient/model/WriteTransaction.java

```diff
@@ -13,21 +13,23 @@
  * permissions and limitations under the License.
  */
 
-package software.amazon.awssdk.extensions.dynamodb.mappingclient.operations;
+package software.amazon.awssdk.extensions.dynamodb.mappingclient.model;
 
 import software.amazon.awssdk.annotations.SdkPublicApi;
 import software.amazon.awssdk.extensions.dynamodb.mappingclient.MappedTableResource;
 import software.amazon.awssdk.extensions.dynamodb.mappingclient.OperationContext;
 import software.amazon.awssdk.extensions.dynamodb.mappingclient.TransactableWriteOperation;
+import software.amazon.awssdk.extensions.dynamodb.mappingclient.operations.PutItem;
 import software.amazon.awssdk.services.dynamodb.model.TransactWriteItem;
 
 /**
- * Encapsulates a single write transaction that can form a list of transactions that go into a TransactWriteItems
- * operation. Example:
+ * Encapsulates a single write transaction that can form a list of transactions that go into a
+ * {@link TransactWriteItemsEnhancedRequest}.
+ * Example:
  *
  * {@code
- * WriteTransaction.of(myTable, putItem(myItem));
- * WriteTransaction.of(myTable, deleteItem(Key.of(stringValue("id123"))));
+ * WriteTransaction.create(myTable, putItem.create(myItem));
```

Comment:
Can we switch this to builder style and remove create implementation? (I don't think it has a future).